### ISSUE TITLE
Airport Revival: Soekarno–Hatta International Airport

### ISSUE THREAD

**Issue by indianbhaji** [openscope/openscope#248]
Soekarno–Hatta International Airport (Indonesian: Bandar Udara Internasional Soekarno–Hatta) (IATA: CGK, ICAO: WIII), abbreviated SHIA or Soetta, is the primary airport serving the Greater Jakarta area on the island of Java, Indonesia. Named after the first president and vice-president of Indonesia, Soekarno and Mohammad Hatta, the airport is located at Benda, Tangerang, approximately 20 km northwest of central Jakarta. Due to its proximity to the suburb of Cengkareng, the airport is also known as Cengkareng Airport, with the assigned three-letter IATA designator: "CGK". The airport is the busiest and the largest airport in Indonesia, as well as being the busiest in the world's southern hemisphere. According to air travel intelligence company OAG, the airport ranked as the 7th most connected airport in the world & ranked 1st as 'megahub' airport in Asia-Pacific region as per connectivity index, ahead of Japan's Tokyo Haneda Airport and Australia's Sydney Airport.

The airport commenced domestic operations in 1985 to replace the old overcapacity Kemayoran Airport, and the airport was expanded by 1991 to replace Halim Perdanakusuma International Airport, for international flights, with Halim only having been commissioned as a temporary solution until Soekarno-Hatta was completed. Kemayoran Airport has since been demolished and is now a residential area. Halim still serves charter, VIP, and private flights, and re-opened as Jakarta's second commercial airport for domestic flights to relieve Soekarno-Hatta which is running over-capacity, due to the rapid growth of airline travel in Indonesia.

The airport often struggles to accommodate all flights at its current limit of 72 planes per hour, frequently forming long lines to take off and for landing. Although over capacity, after a survey from April 23 to May 3 the Airport Council International (ACI) stated on May 4, 2012, that Soekarno–Hatta International Airport is being operated safely. The two existing runways suffer from pavement and strength issues, which limits the airport's capacity to serve large aircraft.To reduce congestion & to achieve a target to handle 100 flights/hour, works for a 3rd runway is currently going on, which is expected to open in 2018. Upgrading of existing two runways are almost completed for safe operation & accommodation of wide-bodied aircraft at the airport.

**Comment by ron-eo** [comment]
Is that why I cannot start the game? I click and it gives me the Preparing Your Shift page but just stays there.

**Comment by indianbhaji** [comment]
@ron-eo No, that's got nothing to do with it. In fact, you won't even see these airports on your airport list. To resolve your issue, try pressing CTRL+F5.

**Comment by ron-eo** [comment]
Hmmm..tried that but no go...just did the refresh and back to the same place. I tried a restart and a power-off/on of my machine but same result.

**Comment by indianbhaji** [comment]
@ron-eo Clear your cache and try again.

**Comment by n8rzz** [comment]
@ron-eo if you are running locally you will need to set up some tooling first. Please see: https://github.com/openscope/openscope/tree/develop/tools for more information on getting things up and running.

**Comment by ron-eo** [comment]
No, trying the web site but may try the local version. Thanx n8rzz!
indianbhaji, I cleared the cache and got "This website is under heavy load

We're sorry, too many people are accessing this website at the same time. We're working on this problem. Please try again later."

Getting very popular because it is a great simulator! :)

**Comment by indianbhaji** [comment]
I'll let @n8rzz handle this

**Comment by ron-eo** [comment]
OK! Thanx!

**Comment by erikquinn** [comment]
Good evening @ron-eo. If you include the `www` prefix, it seems to me to be working.

I get the error when visiting `openscope.co` instead of `www.openscope.co`. We'll work on fixing that.

Created #258 for tracking work on that problem. Thanks for bringing it to our attention :smile:

**Comment by ron-eo** [comment]
So is there any update, folks? I'm still having the same resented to me...Preparing your shift... 

I've tried refresh, clearing the cache, rebooting and power off/on but still the same.

Let me know..am I the only one?

**Comment by n8rzz** [comment]
@ron-eo seems to be working for me?  See the notes in the issue above.  

What browser are you using?

**Comment by ron-eo** [comment]
I am using Firefox v 45.7.0

**Comment by n8rzz** [comment]
can you open the developer tools console and let me know if there are any errors in there?  https://developer.mozilla.org/en-US/docs/Tools/Web_Console/Opening_the_Web_Console

**Comment by ron-eo** [comment]
Good call!

I have: 
TypeError: initialAirportData.icao is undefined
 bundle.min.js:19:22472

under JS

**Comment by n8rzz** [comment]
hmm, that seems like an odd error.  You may have found a bug here.  Lets try one more thing:

try clearing your local storage (which is different than cache but similar to cookies): https://developer.mozilla.org/en-US/docs/Tools/Storage_Inspector

**Comment by ron-eo** [comment]
Ok, I had to enable Storage but, when going to the section, it looks different than what is in the article and, no mater where I right-click, no options come up in a window. Not sure how to clear that storage. :(

**Comment by n8rzz** [comment]
This appears to be a firefox bug.

For the time being, I would suggest running the app in Chrome until this can be worked out.  I've created a new issue for it, it is a legit bug.  So thank you for reporting it!!

**Comment by ron-eo** [comment]
OK, I will try Chrome out. Thanks for all the assistance! :)

**Comment by ron-eo** [comment]
I tried IE first and it was working but seemed to freeze after about 10 planes. I tried Chrome next and it seems to be ok except that is also froze when I went to another browser window. I restarted it and it seems to be working fine. I also noticed that the openscope window doesn't refresh when you take the focus off and go to another web browser window. Thanx!

**Comment by n8rzz** [comment]
@ron-eo lets take the conversation over to #263

**Comment by ron-eo** [comment]
OK...sorry!

Thanx!
